jsbridge/src/main/jni/JavaTypeProvider.cpp

```diff
@@ -65,10 +68,13 @@ const JavaType *JavaTypeProvider::newType(const JniRef<jsBridgeParameter> &param
     case JavaTypeId::Unknown:
       return nullptr;
     case JavaTypeId::Void:
+      return new Void(m_jsBridgeContext, id, false /*boxed*/);  // TODO: check it
```

Comment:
Should this Todo be fixed before merging? Same for [another one](https://github.com/p7s1digital/oasis-jsbridge-android/pull/25/files#diff-43450cc29dc8b1056120b7b4eabd19bcf9fe10e598c57242e4394cf87bbe685eR138) down the code.

Reply:
Thanks for pointing it out! The change has already been checked and removed the comment. The other one way still requiring a change which has now been addressed :)